src/utilities/dockerUtils.ts

```diff
@@ -24,6 +24,9 @@ export async function getDeploymentConfig(): Promise<DeploymentConfig> {
    }
 
    const imageNames = core.getInput('images').split('\n') || []
+   if (imageNames.length == 1 && imageNames[0] == '') {
```

Comment:
this doesn't fix the edge case. what if the input is "  "? We would still want an empty array for input.